golang/cosmos/types/events.go

```diff
@@ -19,23 +19,20 @@ const (
 	AnchoredKeyEnd   = AnchoredKeyStart
 )
 
-func NewStateChangeEvent(storeName string, subkey, value []byte) sdk.Event {
+func NewStateChangeEvent(storeName string, subkey []byte, value string) sdk.Event {
 	// This is a hack to allow CONTAINS event queries to match the
 	// beginning or end of the subkey, enabling a handy OR of changes to a
 	// vstorage path's immediate children.
 	//
 	// FIXME: add string value ranges to Tendermint event queries instead.
-	anchoredKey := bytes.Join([][]byte{
-		[]byte(AnchoredKeyStart),
-		subkey,
-		[]byte(AnchoredKeyEnd),
-	}, []byte{})
+	subkeyStr := string(subkey)
```

Comment:
Do you mean the following?
```suggestion
	subkeyStr := conv.UnsafeBytesToStr(subkey)
```
If not, maybe say why not, like in some "invariant" comments attached to the functions in `conv`?